### ISSUE TITLE
earthquake

### ISSUE THREAD

**Issue by Mariusthvdb** [Templarian/MaterialDesign#4358]
I have:

- [ ] [Searched all issues](https://github.com/Templarian/MaterialDesign/issues) to make sure there isn't a request for this icon.
- [ ] [Searched the current library](https://materialdesignicons.com/) to make sure the icon doesn't exist.
- [ ] Only requested a single icon (or a few near-identical ones) in this issue. 

## Usage

Provide some context of how your icon could be used.

need an icon for the earth quake feed sensor, in case of active quakes (or none..)

## Examples

Include any example images so we know what the icon should look like.

now use the

```
        icon_template: >
          {% if states('sensor.count_earthquakes')|int != 0 %} mdi:google-earth
          {% else %} mdi:earth
          {% endif %}
```
because the google-earth is a somewhat twisted Earth ;-)

preferably we would have an earth with a crack in it maybe, or, taking it down the seismic road, use a graph of some sort, but that might be too complex for such a small icon?

thanks for considering

**Comment by MrGrigri** [comment]
Would this suffice?

![image](https://user-images.githubusercontent.com/4528551/63532369-c427f600-c4c7-11e9-935d-ddb06732ff87.png) ![image](https://user-images.githubusercontent.com/4528551/63532404-d4d86c00-c4c7-11e9-8aa4-02c36ddcb4f0.png)

---

#### Files

[home-broken.zip](https://github.com/Templarian/MaterialDesign/files/3531061/home-broken.zip)

**Comment by Mariusthvdb** [comment]
o dear that looks rather horrific ;-) might be a bit too much for a warning....seems to be in the fatal stage already.

I do see use for these, but had hoped for an icon with the earth featuring in it.

**Comment by JamesCoyle** [comment]
Could always do earth-vibrate :P

**Comment by MrGrigri** [comment]
![image](https://user-images.githubusercontent.com/4528551/63553812-dff7c000-c4f8-11e9-901b-32fad06c81bc.png)

---
#### Files
[earth-vibrate.zip](https://github.com/Templarian/MaterialDesign/files/3532125/earth-vibrate.zip)

**Comment by PeterShaggyNoble** [comment]
Hmm ... Dunno that that would communicate earthquake to me, even with context.

Would an `alert` modifier work with the `earth` icon as a generic "disaster" icon?

**Comment by MrGrigri** [comment]
![image](https://user-images.githubusercontent.com/4528551/63555672-146e7a80-c4ff-11e9-8692-7ee8df353992.png)

---
#### Files

[earth-alert.zip](https://github.com/Templarian/MaterialDesign/files/3532218/earth-alert.zip)

**Comment by JapanYoshi** [comment]
I looked up "earthquake icon" in Japanese (Japan has a lot of earthquakes so I thought we'd have a better icon) and most of them were either of a shaking building, a cracking building (as in @MrGrigri's first suggestion), or some combination.
![image](https://user-images.githubusercontent.com/8707990/63556035-9655b680-c586-11e9-9abf-854d77a1cc69.png)

**Comment by MrGrigri** [comment]
What about a `house-pulse`?

![image](https://user-images.githubusercontent.com/4528551/63556891-b09a8080-c503-11e9-8235-ceeead47708a.png)

---
### Files
[house-pulse.zip](https://github.com/Templarian/MaterialDesign/files/3532283/house-pulse.zip)

**Comment by JapanYoshi** [comment]
Even after shrinking the base of the `home` icon by 1dp on each side, the vibration lines are hard to include. We'll need to use a smaller version.

With 2dp lines:
![image](https://user-images.githubusercontent.com/8707990/63728447-0a55ce80-c8a7-11e9-9791-778e72d7964e.png)
With 1½dp lines:
![image](https://user-images.githubusercontent.com/8707990/63728452-117cdc80-c8a7-11e9-8f2d-79dd3b6b002a.png)

**Comment by Mariusthvdb** [comment]
tbh, I don't think this has the looks of a quake at all, seems more like a fence to me. If you want too stick with the house as main icon for earthquake (instead of the earth..), I suppose one of your earlier posts has a better one:

![Schermafbeelding 2019-08-27 om 08 44 39](https://user-images.githubusercontent.com/33354141/63747325-278a9d00-c8a7-11e9-93e3-ede0d86cfa4a.png)

main visuals: the crack in the earth (!) and the trembling curved lines along the house.

**Comment by Mariusthvdb** [comment]
has anything been decided on this icon? if anything, home-broken in the first answer would already be appreciated

**Comment by Mariusthvdb** [comment]
dear people, contributors to my issue..

I would love to add these icons you made to my Custom Icons repo at https://github.com/Mariusthvdb/Custom-icons.
before releasing that, I kindly request your permission. Ofc full credits will be given.

@JapanYoshi @MrGrigri

**Comment by JapanYoshi** [comment]
Out of these suggestions, I would go for [`house-broken`](https://github.com/Templarian/MaterialDesign/issues/4358#issuecomment-523979769),

**Comment by Bagunda** [comment]
Here is a good icon that is already in [one of the home assistant modules](https://www.home-assistant.io/integrations/usgs_earthquakes_feed/) (this would be added):

![usgs-earthquake-hazards-program-feed-entities](https://user-images.githubusercontent.com/16766521/219972630-ff6880c9-f04c-49e1-9cdf-7bb873cda23b.png)

**Comment by Mariusthvdb** [comment]
yes, that is https://pictogrammers.com/library/mdi/icon/pulse/ as is used in house-pulse above

**Comment by Mariusthvdb** [comment]
btw, guess this can be closed as no official icon will be added?
please any of the devs in charge feel free to do so?

**Comment by JapanYoshi** [comment]
> Here is a good icon that is already in [one of the home assistant modules](https://www.home-assistant.io/integrations/usgs_earthquakes_feed/) (this would be added):
> 
> ![usgs-earthquake-hazards-program-feed-entities](https://user-images.githubusercontent.com/16766521/219972630-ff6880c9-f04c-49e1-9cdf-7bb873cda23b.png)

> yes, that is https://pictogrammers.com/library/mdi/icon/pulse/ as is used in house-pulse above

Alias that icon as `earthquake` and `seismometer`?
